src/_data/side-nav.yml

```
@@ -212,7 +201,13 @@
       - title: Dart DevTools
         permalink: /tools/dart-devtools
       - title: DartPad
-        permalink: /tools/dartpad
+        children:
+          - title: Overview
+            permalink: /tools/dartpad
```

You'll need to add these to the "Overview" entry, so that the other pages show up as active when open

```suggestion
          - title: Overview
            urlExactMatch: true
            match-page-url-exactly: true
            permalink: /tools/dartpad
```

---

Do you mean so they'll show as blue/selected in the side nav? That's what I'm observing, just want to confirm

---

Yep :) otherwise only **Overview** will show up as blue/highlighted as it is located at a parent location.